pkg/registration/register.go

```diff
@@ -1,18 +1,73 @@
 package registration
 
 import (
-	"github.com/SUSE/connect-ng/internal/collectors"
+	"encoding/json"
+
 	"github.com/SUSE/connect-ng/pkg/connection"
 )
 
+type announceRequest struct {
+	Hostname          string `json:"hostname"`
+	SystemInformation any    `json:"hwinfo"`
+}
+
+type announceResponse struct {
+	Id       int    `json:"id"`
+	Login    string `json:"login"`
+	Password string `json:"password"`
+}
```

Comment:
This feels like something that should be exported.

Reply:
Why? What is the benefit of it reporting it? Storing the `id` somewhere?

Reply:
Ah, you are right, nevermind.